app/controllers/api/v2/courses_controller.rb

```diff
@@ -47,27 +47,34 @@ def publish
       end
 
       def update
+        update_enrichment if enrichment_params.values.any?
+        update_sites
```

Comment:
Should this be
```suggestion
        update_sites if params[:course][:sites_ids].present?
```
for symmetry?

Reply:
Yeah, was thinking about putting that in its own `sites_params` method 👍 I'll update

Reply:
Nevermind; this makes codeclimate unhappy. Have used a guard clause instead.